sound/soc/sof/intel/ext_manifest.h

```diff
@@ -0,0 +1,34 @@
+/* SPDX-License-Identifier: ((GPL-2.0 WITH Linux-syscall-note) OR BSD-3-Clause) */
+/*
+ * This file is provided under a dual BSD/GPLv2 license.  When using or
+ * redistributing this file, you may do so under either license.
+ *
+ * Copyright(c) 2020 Intel Corporation. All rights reserved.
+ */
+
+/*
+ * Intel extended manifest is a extra place to store Intel cavs specific
+ * metadata about firmware, for example LPRO/HPRO configuration is
+ * Intel cavs specific. This part of output binary is not signed.
+ */
+
+#ifndef __INTEL_CAVS_EXT_MANIFEST_H__
+#define __INTEL_CAVS_EXT_MANIFEST_H__
+
+#include <sound/sof/ext_manifest.h>
+
+/* EXT_MAN_ELEM_PLATFORM_CONFIG_DATA elements identificators */
+enum sof_cavs_config_elem_type {
+	SOF_EXT_MAN_CAVS_CONFIG_EMPTY		= 0,
```

Comment:
what is the point of an EMPTY token that will be skipped anyways?

Reply:
@plbossart I think the explanation here was to check against 0-initialised data, as used in `switch` below, but I also don't find this neither useful nor common. There are *some* cases in the kernel where `enum`s skip 0 and begin with 1, but usually there's a very specific reason for that. Also the naming of `LAST_ELEM` which isn't in fact the last element but the one after it, usually named `NUMBER_OF_ELEMENTS` or similar. And using it as the size of an array in the ABI - all this I still don't like in this (and related) patch(es).

Reply:
We need explicit EMPTY token to skip, I think, this is better than just comparing 0.
And also this is symmetrical change include/sound/sof/ext_manifest.h

https://github.com/thesofproject/linux/blob/topic/sof-dev/include/sound/sof/ext_manifest.h#L105
```
	SOF_EXT_MAN_CONFIG_EMPTY		= 0,
```